### ISSUE TITLE
Fix extent.yaml schema error

### ISSUE THREAD

**Issue by m-burgoyne** [opengeospatial/ogcapi-environmental-data-retrieval#401]
The [extent.yaml](https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/blob/master/standard/openapi/schemas/collections/extent.yaml) has an indentation error in the bbox definition

```yaml
    oneOf:
      - items:
          minItems: 4
          maxItems: 4
          type: number
        type: array  
      - items:
          minItems: 6
          maxItems: 6
          type: number
        type: array  
```

should be
```yaml
          oneOf:
            - items:
                type: number
              minItems: 4
              maxItems: 4
              type: array  
            - items:
                type: number
              minItems: 6
              maxItems: 6
              type: array  
```

**Comment by m-burgoyne** [comment]
@chris-little both

**Comment by chris-little** [comment]
@ghobona We would like to correct this schema fragment in V1.0.1. Can we do it easily? Is there an impact on the CITE tests?

**Comment by ghobona** [comment]
@chris-little The fix can be done editorially on the V1.0.1 document. I'll create a ticket for the Infrastructure Office to update the document.

We'll also check and update the ETS on the next release cycle.

**Comment by ghobona** [comment]
@m-burgoyne Please hold, before updating the `1.0.1` branch.

@chris-little We have feedback that the change would require a bug fix to `1.0.2` because it relates to the schema. Might the SWG be interested in developing a `1.0.2` ?

I ask this question because I know that the current focus of the SWG is on Version `1.1.0`. So there might be limited benefit in issuing a `1.0.2`. Comments are welcome.

**Comment by chris-little** [comment]
@ghobona We would rather focus on V1.1, as it is meant to be out for public review. We can raise a formal comment and create a v1.0.2 later.

**Comment by chris-little** [comment]
V1.1 now published, which replaces V1.0.1, so this issue can be closed.
